internal/config/config.go

```diff
@@ -1,14 +1,15 @@
 package config
 
 import (
-	"errors"
```

Comment:
For open source I'd encourage us to stick with what the broader community is already using, and these days that'd be the stdlib package. I'm not sure stack traces would be super helpful in a CLI app.

Reply:
@mdlayher We're already using error wrapping in a bunch of other places and would prefer to keep that consistent then. I think the value for us more in the wrapping than in stack traces.

Reply:
Oh I see now, this is already in use in other files in the repo. SGTM.